### ISSUE TITLE
Obsolete theme

### ISSUE THREAD

**Issue by sollago** [marlago/website#32]
- In mobile: why is there so much space under the footer? In PC, it looks ok, though, so I don't know why the problem only affects mobile.

**Comment by iliakur** [comment]
@sollago I think you can close this now.

**Comment by sollago** [comment]
@Copper-Head but the problem still occurs

**Comment by iliakur** [comment]
this is coming from deep inside the theme, some javascript nonsense. I was also alarmed to discover that the framework on which our theme is built hasn't been updated in 2 years!

This to me says that we should start looking for a theme that builds on a more robust foundation.
It would also be nice to find something that less complicated than the theme we currently use. It works, but is a bit of a mess :)

**Comment by sollago** [comment]
Good idea! Let me look at other theme so we can implement this when there is more time.

**Comment by sollago** [comment]
Ok, since we are meeting soon, I made a list of possible themes/websites whose styles I like:

- https://people.umass.edu/bwdillon/

- http://faculty.wcas.northwestern.edu/matt-goldrick/v2/#!/

- http://www.ladlab.com/
